packages/internals/src/engine-commands/getConfig.ts

```diff
@@ -83,6 +83,12 @@ ${detailsHeader} ${message}`
   }
 }
 
+export function getDirectUrl(ds: DataSource): EnvValue {
```

Comment:
Nit:

This confused me a bit: Why is the method that 99% returns the `url` and only sometimes the `directUrl` called `getDirectUrl`?

But I think I understand now: Either `url` or `directUrl` are the _direct URL_ we want to use. Is that your motivation for this naming?

As "direct URL" and "`directUrl`" are the same term, I think I might name the function `getEffectiveUrl` or something like that.

Reply:
A rename would be nice to avoid confusion, can be done later if needed